dev-env/bin/dade-copyright-headers

```diff
@@ -60,6 +60,7 @@ filetypes = \
     'tf'     : { 'start' : '#' * 80 , 'line' : '#'    , 'end' : ''    , 'empty_line' : '' } ,
     'css'    : { 'start' : '/*'     , 'line' : ''     , 'end' : ' */' } ,
     'scss'   : { 'start' : '/*'     , 'line' : ''     , 'end' : ' */' } ,
+    'yml'    : { 'start' : '#' * 80 , 'line' : '#'    , 'end' : ''    , 'empty_line' : '' } ,
```

Comment:
Should we also match on `yaml`?

Reply:
Good point, will add that.

Reply:
We have a bunch of `.yaml.template` files, but I imagine we do not want copyright headers on those. 🤔